src/testRunner/parallel/host.ts

```diff
@@ -525,7 +525,7 @@ namespace Harness.Parallel.Host {
                 completeBar();
                 progressBars.disable();
 
-                const replayRunner = new Mocha.Runner(new Mocha.Suite(""), /*delay*/ false);
+                const replayRunner = new Mocha.Runner(new Mocha.Suite(""), { delay: false });

```

Comment:
If you do a fresh `npm install` then you get runtime warnings about this

Reply:
This was deprecated officially in mocha 9, released in June, and has has the object form listed as an option for at least a few major versions longer - so the change seems fine to me~